microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -121,79 +152,38 @@ As the recording progresses, the JS heap size spikes are displayed.  This is nat
-```
-
-Clicking the button referenced in the code creates a `ul` node with ten `li` children.  The nodes are referenced by the code, but they don't exist in the DOM tree, so each node is detached.
-
-Heap snapshots are one way to identify detached nodes.  As the name implies, heap snapshots show you how memory is distributed among the JS objects and DOM nodes for your page at the point of time of the snapshot.
-
-To create a snapshot:
-
-1. Open DevTools and go to the **Memory** tool.
-
-1. Click the **Heap snapshot** radio button, and then click the **Take snapshot** button at the bottom of the tool.
-
-   ![Taking a heap snapshot](./index-images/memory-heap-snapshot.png)
-
-   The snapshot may take some time to process and load.
-
-1. After the snapshot is finished, select it from the left-hand panel (it's named **HEAP SNAPSHOTS**).
-
-1. In the **Class filter** text box, type `Detached`, to search for detached DOM trees:
-
-   ![Filtering for detached nodes](./index-images/memory-heap-snapshot-filter-detached.png)
-
-1. Expand the carats to investigate a detached tree:
-
-   ![Investigating the detached tree](./index-images/memory-heap-snapshot-filter-detached-expanded.png)
-
-   <!--
-   Nodes that are highlighted yellow have direct references to them from the JavaScript code.  Nodes that are highlighted in red don't have direct references.  They are only alive because they are part of the tree for the yellow node.  In general, you want to focus on the yellow nodes.  Fix your code so that the yellow node isn't alive for longer than it needs to be, and you also get rid of the red nodes that are part of the tree for the yellow node.
-   -->
-
-1. Click a node to investigate it further.
-
-   In the **Objects** pane, you can see more information about the code that is referencing the node.  For example, in the following figure, the `detachedTree` variable is referencing the node.
-
-1. To fix the particular memory leak, study the code that uses the `detachedTree` variable and make sure that the reference to the node is removed when it is no longer needed.
-
-![Investigating a node](./index-images/memory-heap-snapshot-filter-detached-expanded-selected.png)
+<!--[1]-->
+<!-- ====================================================================== -->
+## Record a heap snapshot ("Heap snapshot" profiling type)
 
-<!--todo: the allocation timeline doesn't appear in the DevTools in Edge  -->
+See:
+* [Record heap snapshots using the Memory tool ("Heap snapshot" profiling type)](./heap-snapshots.md).
+* [Find DOM tree memory leaks ("Heap snapshot" profiling type > Detached)](#find-dom-tree-memory-leaks-heap-snapshot-profiling-type--detached), below.
 
 
+<!--[2]-->
 <!-- ====================================================================== -->
-## Identify JS heap memory leaks with Allocation instrumentation on timeline
+## Identify JS heap memory leaks ("Allocations on timeline" profiling type)
```

Comment:
move content into article:
"Use Allocation instrumentation on timeline ("Allocations on timeline" profiling type)"
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler?branch=pr-en-us-3282

present article is too long for LMC, and dup of section + article is confusing for maintainers & users

Reply:
status: 
Moved h2 section's content to the dedicated article, allocation-profiler.md:
**Use Allocation instrumentation on timeline ("Allocations on timeline" profiling type)**

that article has some explanation and then a UI steps section showing updated Mem tool home page - not sure if meets my design closely.  

Target design:
* Mem tool: Very short article, just tables of contrasting overviews: Use tool x to do ABC (link to article).
* Article about one of the profiling types: 1) Use this prof type to ABC.  2) UI steps.  3) verbose details.

Mem tool article status here: kept the h2 heading / section , now empty except for a link.  Linked to dedicated article (not to h2 section) in table at top.  I'll probably end up deleting this now-empty h2 section, but not quite yet bc need h2 section about each prof type, until Mem tool article is emptied out and changed to pure overview/contrast/nav.

@captainbrosset